src/Statistics.jl

```diff
@@ -175,9 +300,10 @@ function var(
         m = mean(M, x; kwargs...)
     end
     n = length(x)
-    w = ProbabilityWeights(ones(n), n)
+    w = _var_weights(n)
```

Comment:
what's `_var_weights`? I might have missed it...

Reply:
`_var_weights` and `_mean_weights` were utility functions to create unit weights of length `n`, but they've now been merged to `_unit_weights`. When the next version of StatsBase is released, we can make it return `StatsBase.UnitWeights`.